xml/System.Threading.Tasks/Task.xml

```diff
@@ -5447,10 +5447,26 @@ Task t Status: RanToCompletion, Result: 42
   
 ## Remarks  
  You can use `await Task.Yield();` in an asynchronous method to force the method to complete asynchronously. If there is a current synchronization context (<xref:System.Threading.SynchronizationContext> object), this will post the remainder of the method’s execution back to that context. However, the context will decide how to prioritize this work relative to other work that may be pending. The synchronization context that is present on a UI thread in most UI environments will often prioritize work posted to the context higher than input and rendering work. For this reason, do not rely on `await Task.Yield();` to keep a UI responsive.  For more information, see the entry [Useful Abstractions Enabled with ContinueWith](http://blogs.msdn.com/b/pfxteam/archive/2008/07/23/8768673.aspx) in the Parallel Programming with .NET blog.  
+
+A good example of the usage would be a `Windows Forms` application where you would like to execute a lengthy operation, but at the same time release the thread so the UI would be accessible to user. This operation will get run at a later time (at which point, it still may run synchronously) on the current context.
```

Comment:
@yashints thanks for the update. A question, though:
>...get run at a later time (at which point, it still may run synchronously)...

The statement is the parentheses contradicts to the first sentence in the **Remarks** section:
>...to force the method to complete asynchronously...

I still think that the comment in the code:
>// unblock the UI before doing potentially lengthy operation

is misleading. Yes, we return to the caller after `await Task.Yield();`, but then, eventually, UI thread will be occupied with `LoadLocalConfiguration` execution. So, we block UI anyway, but later.

Also, the question: why not to use
```csharp
await Task.Run(() => LoadLocalConfiguration());
``` 

instead of 
```csharp
await Task.Yield();  // unblock the UI before doing potentially lengthy operation
LoadLocalConfiguration();
```

Reply:
Agree with the first point, in terms of using `Task.Yeild()` vs `Task.Run()` that was to let the user know what would happen more explicitly. Happy to change if needed tho

Reply:
I agree with @pkulikov 's suggestions above.  

Also, I apologize for letting this get stale without a response.  I've resolved the conflicts. @yashints If you make the changes suggested above, I'll :shipit: today.

Reply:
Resolved, did some minor changes to wording as well